tms/clickup/views.py

```diff
@@ -19,3 +23,35 @@ def post(self, request):
         return JsonResponse({'token': token.key,
                              "user": user.username,
                              "message": "Successfully logged in."}, status=status.HTTP_200_OK)
+
+
+class ProjectCreateView(ListCreateAPIView):
+    queryset = Project.objects.all()
+    serializer_class = ProjectSerializer
+
+    def post(self, request, *args, **kwargs):
```

Comment:
Why override this message?

Reply:
because of i not override methods that time response in only data not message so i over all methods.